Need to upgrade the related gRPC dependencies.

Hi, I have met some grpc errors when I using `dgraph4j: 2.0.2` and `protobuf-java: 3.7.1`.

The error:
```
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: Connection closed while performing protocol negotiation for [HttpClientCodec#0, HttpProxyHandler#0, ProtocolNegotiators$WaitUntilActiveHandler#0, WriteBufferingAndExceptionHandler#0, DefaultChannelPipeline$TailContext#0]
```

And the solution is to exclude the old version of grpc dependencies in `dgraph4j` and added the new versions. For `dgraph4j: 2.0.2` version, we could do this at `pom.xml`: 

```
...
    <dependency>
      <groupId>com.google.protobuf</groupId>
      <artifactId>protobuf-java</artifactId>
      <version>3.11.1</version>
    </dependency>

    <dependency>
      <groupId>io.dgraph</groupId>
      <artifactId>dgraph4j</artifactId>
      <version>2.0.2</version>
      <exclusions>
        <exclusion>
          <groupId>io.grpc</groupId>
          <artifactId>grpc-protobuf</artifactId>
        </exclusion>
        <exclusion>
          <groupId>io.grpc</groupId>
          <artifactId>grpc-stub</artifactId>
        </exclusion>
        <exclusion>
          <groupId>io.grpc</groupId>
          <artifactId>grpc-netty</artifactId>
        </exclusion>
      </exclusions>
    </dependency>

    <dependency>
      <groupId>io.grpc</groupId>
      <artifactId>grpc-protobuf</artifactId>
      <version>1.26.0</version>
    </dependency>
    <dependency>
      <groupId>io.grpc</groupId>
      <artifactId>grpc-stub</artifactId>
      <version>1.26.0</version>
    </dependency>
    <dependency>
      <groupId>io.grpc</groupId>
      <artifactId>grpc-netty</artifactId>
      <version>1.26.0</version>
    </dependency>

...

```

The detail info about the issue: https://discuss.dgraph.io/t/dgraph4j-using-java-client-connect-to-dgraph-alpha-failed-docker-on-aws-ec2/5829

I think we could upgrade the related dependencies in `dgrap4j` to avoid this same problems.

Thanks.

--- a/build.gradle
+++ b/build.gradle
@@ -45,7 +45,7 @@ version = '2.1.0'
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
-def grpcVersion = '1.22.1'
+def grpcVersion = '1.26.0'
 def dgraph4jVersion = "$version"
 def openCensusVersion = '0.23.0'
 

